Tests/DependencyInjection/Fixtures/yml/server_log.yml

```diff
@@ -0,0 +1,5 @@
+monolog:
+    handlers:
+        server_log:
+            type:       server_log
+            host:       0:9911
```

Comment:
Please add this as an example on top of the Configuration class (in the class docblock we have examples of all handlers)